README.md

```diff
@@ -4,4 +4,7 @@
 
 ## License
 
-MIT © [AgFlow](https://www.agflow.com)
+MIT © [AgFlow](https://www.agflow.com) 
+
+###
```

Comment:
this token shouldn't be needed, right ?

Reply:
Yes, but just took that for like separator. If its gonna be problem I can remove it instantly

Reply:
not a problem but just extra